modules/nodes-pods-configmap-creating-from-directories.adoc

```diff
@@ -57,39 +75,18 @@ allow.textmode=true
 how.nice.to.look=fairlyNice
 ----
 
+.Procedure
+
 . Create a config map holding the content of each file in this directory by entering the following command:
```

Comment:
Should this be a bullet since it is a one step procedure? I see the verification step as well and they are both step 1. I am not familiar with if that is how we should format or not. I guess they could both be bullets too.